packages/kbn-apm-synthtrace/src/scenarios/mobile.ts

```diff
@@ -354,34 +364,40 @@ const scenario: Scenario<ApmFields> = async ({ scenarioOpts, logger }) => {
           device.startNewSession();
```

Comment:
not related to this PR, but synthtrace shouldn't have side-effects like this. We should use an API like `.session()` which returns a session. can you add a FIXME comment to the `startNewSession` method?